odoo/addons/base/views/res_company_views.xml

```diff
@@ -71,6 +71,7 @@
                     <field name="name"/>
                     <field name="partner_id"/>
                     <field name="child_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
+                    <field name="active" invisible="True"/>
```

Comment:
And this doesn't seem to be in the right commit

Reply:
Oops, should be corrected now